packages/plugin-ext/src/common/plugin-protocol.ts

```diff
@@ -24,6 +24,7 @@ import { RecursivePartial } from '@theia/core/lib/common/types';
 import { PreferenceSchema, PreferenceSchemaProperties } from '@theia/core/lib/common/preferences/preference-schema';
 import { ProblemMatcherContribution, ProblemPatternContribution, TaskDefinition } from '@theia/task/lib/common';
 import { ColorDefinition } from '@theia/core/lib/browser/color-registry';
+import { ResourceLabelFormatter } from '@theia/core/lib/browser/label-provider';
```

Comment:
please use `import type` here to make sure that we don't pull runtime deps from browser to common, to fix it cleanly we should move `ResourceLabelFormatter` to common in core

Reply:
moved `ResourceLabelFormatter` to common in core